src/administration/integrations/tideways.md

```diff
@@ -38,6 +41,13 @@ git push
 
 Tideways should now be enabled.  Give it a few hours to a day to get a decent set of data before checking your Tideways dashboard.
 
-## Email integration
+## Deployment Integration
 
-Once Tideways is enabled, an additional environment variable will be available.  The `PLATFORM_SMTP_HOST` variable will contain the hostname of the Tideways SMTP server should it be needed.
+Tideways integrates with platform.sh Deployment Hooks and provides performance comparisons
```

Comment:
```suggestion
Tideways integrates with Platform.sh deployment hooks and provides performance comparisons
```

Reply:
This one was redundant, already applied :smile: